enterprise_access/apps/content_assignments/models.py

```diff
@@ -283,6 +283,31 @@ def add_successful_notified_action(self):
         )
         return record, was_created
 
+    def get_successful_cancelled_action(self):
+        """
+        Returns the first successful "cancelled" LearnerContentAssignmentActions for this assignment,
+        or None if no such record exists.
+        """
+        return self.actions.filter(
+            action_type=AssignmentActions.CANCELLED,
+            error_reason=None,
+        ).first()
+
+    def add_successful_cancelled_action(self):
```

Comment:
Hmm, did you mean to use this in your task to create a new cancelled action for the assignment?

Reply:
Removed this since I don't think we'll need it.